### PR TITLE
tests: run sg example & update readme

### PR DESCRIPTION
**Pull request by jor2**
### Description

Replace this text with a summary of the changes in this PR. Include why the changes are needed and context about the changes. List required dependencies. If there is a Git issue for the change, please link to it.

#### No release required

- [x] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other changes that don't affect Terraform code

#### Release required

- [ ] Bug fix (patch release (`x.x.X`): Change that fixes an issue and is compatible with earlier versions)
- [ ] New feature (minor release (`x.X.x`): Change that adds functionality and is compatible with earlier versions)
- [ ] Breaking change (major release (`X.x.x`): Change that is likely incompatible with previous versions)

##### Release notes content

Replace this text with information that users need to know about the bug fixes, features, and breaking changes. This information helps the merger write the commit message that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Merge by using "Squash and merge".
- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author.

    The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).

### CODE REVIEW

**Comment by jor2** [comment]
> @jor2 - pre-commit checks are not passing in the GH action.

Fixed, its because in the go.mod file the precommit hooks keeps correcting it to `module github.com/terraform-ibm-modules/terraform-ibm-icse-vpe-module` instead of `module github.com/terraform-ibm-modules/terraform-ibm-vpe-module`. Any idea how I should fix this autocorrect?

File: README.md [hunk 1]
```diff
@@ -1,135 +1,79 @@
...
-### Post-merge steps
-
-After the first PR for your module is merged, follow these post-merge steps:
-
-1.  Create a PR to enable the upgrade test by removing the `t.Skip` line in `tests/pr_test.go`.
-
-<!-- Remove the content in this previous H2 heading -->
+The module supports creating and updating settings that are applied with the `terraform apply` command. With objects
+affected by the `destroy` command, the module preserves the most recent setting and doesn't change objects that are
+configured outside of Terraform's scope.
```
Comment (SirSpidey) [inline README.md:16]:
```suggestion
With this module you can create and update settings that are applied with a `terraform apply` command. For objects
that are affected by a `terraform destroy` command, the module preserves the most recent setting, and doesn't change objects that are configured outside of Terraform's scope.
```

**Review by SirSpidey — DISMISSED** [review]
Some suggestions to match with Cloud terms and IBM style

File: README.md [hunk 2]
```diff
@@ -1,135 +1,79 @@
...
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
-[![latest release](https://img.shields.io/github/v/release/terraform-ibm-modules/terraform-ibm-module-template?logo=GitHub&sort=semver)](https://github.com/terraform-ibm-modules/terraform-ibm-module-template/releases/latest)
+[![latest release](https://img.shields.io/github/v/release/terraform-ibm-modules/terraform-ibm-vpe-module?logo=GitHub&sort=semver)](https://github.com/terraform-ibm-modules/terraform-ibm-vpe-module/releases/latest)
 
-<!-- Remove the content in this H2 heading after completing the steps -->
+This module can be used to create and configure an [Endpoint Gateway](https://cloud.ibm.com/docs/vpc?topic=vpc-ordering-endpoint-gateway&interface=ui).
 
-## Submit a new module
+The module supports the following scenarios:
+- Create Reserved IPs, Endpoint Gateways, and Attach Endpoint Gateways to Reserved IPs.
```
[the remark this answers is not in the document]
Comment (vburckhardt) [reply README.md:12]: I think actions is fine here.

**Review by vburckhardt — COMMENTED** [review]
Thanks @jor2 - some comments.

**Review by vburckhardt — APPROVED** [review]

**Pull request merged**